Project/DAQ_System/Program/Src/Sensor/LinearPotentiometer/ilinear_potentiometer.hpp

```diff
@@ -20,8 +20,8 @@ class ILinearPotentiometer {
     public:
         virtual ~ILinearPotentiometer() = default;
 
-        virtual float DisplacementInches() = 0;
-        virtual float DisplacementMillimeters() = 0;
+        virtual void DisplacementInches(float data[]) = 0;
+        virtual void DisplacementMillimeters(float data[]) = 0;
```

Comment:
Perhaps name the array within the format of the data being received. I liked the name `quantized_counts` — something more explicit.

Reply:
imma take that variable name from ya